### DISCUSSION TITLE
Markdown Heading - Fold and Unfold

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#111] [Feature Requests]
[Arjun]: Current behavior - All headings are rendered as it is
Expected behavior - All headings level 1-6 should have marker/handle to fold-unfold to hide-see content under specific heading for better page viewing.

**Comment by black750wars** [comment]
Cool feature to have
But i would love this more:
A button which will move from this heading to the next one
Even if both headings are level 2it will move to the next heading

Think of it like a website which have a step by step guide

I can view all steps (heading) then move easily to the point I want

**Reply by damionx7** [reply]
You should try the outline features in v2.9.5. It lists all the headings and you can navigate by clicking on each heading.
